docker-compose.yml

```diff
@@ -38,50 +40,12 @@ services:
       - PHYSICAL_ADDRESS_ANNOTATOR_PORT=8080
     networks:
       - nlp-sandbox-internal
+      - nlp-sandbox
     depends_on:
       - date-annotator
       - person-name-annotator
       - physical-address-annotator
 
-  phi-deidentifier-app:
-    image: nlpsandbox/phi-deidentifier-app:edge
-    build:
-      context: client
-      dockerfile: Dockerfile
-    container_name: phi-deidentifier-app
-    environment:
-      - SERVER_PORT=${SERVER_PORT}
-      - SERVER_PROTOCOL=${SERVER_PROTOCOL}
-      - SERVER_HOST=${SERVER_HOST}
-      - SERVER_API_PATH=${SERVER_API_PATH}
-    restart: always
-    networks:
-      - nlp-sandbox-internal
-    depends_on:
-      - phi-deidentifier
-
-  nginx:
```

Comment:
Keep the nginx service so that the date annotator is isolated and can not communicate with the internet. This is important for 1) security reason and 2) this tool should be able to work without access to internet the day we start accepting submission to benchmark "phi-deidentifier" tools.

Use the `docker-compose.yml` of the Date Annotator as example: https://github.com/nlpsandbox/date-annotator-example. You should be able to copy/paste the nginx folder of the Date Annotator and use it as it is.

Reply:
Don't these lines:

```yaml
    networks:
      - nlp-sandbox-internal
```

prevent all annotators from accessing the outside network (i.e. the internet)?

Reply:
OK